src/createRouter.ts

```diff
@@ -109,6 +112,12 @@ export interface RouterFactory extends ComposeFactory<Router, RouterOptions> {
 	(options?: RouterOptions): Router;
 }
 
+function diffHeirarchy(a: Selection[], b: Selection[]): Selection[] {
```

Comment:
again I think you're right, i don't think we should call this hierarchy at all :smile: